tests/integration/conftest.py

```diff
@@ -46,7 +47,12 @@ async def charmed_kubernetes(ops_test):
             deploy, control_plane_app = False, control_plane_apps[0]
 
         if deploy:
-            await model.deploy("kubernetes-core", channel="latest/edge")
```

Comment:
what was the issue with libjuju?

Reply:
There seems to be an [issue with python-libjuju](https://chat.charmhub.io/charmhub/pl/na6nde1os38b8ki19f5ek4b49w) which was discussed with @hmlanigan but it probably needs a more formal issue tracked in project

Reply:
https://github.com/juju/python-libjuju/issues/766